src/ttsim/interface_dag_elements/results.py

```diff
@@ -17,15 +17,18 @@
 
 
 @interface_function()
-def tree(raw_results__combined: QNameData) -> NestedData:
+def tree(raw_results__combined: QNameData, input_data__tree: NestedData) -> NestedData:
```

Comment:
```suggestion
def tree(raw_results__combined: QNameData, input_data__flat: FlatData) -> NestedData:
```
Reverse of above. Then use `path = dt.tree_path_from_qual_name(k)` inside the loop in order to use the same conditions as there.